styleguide/writing-style.md

```diff
@@ -0,0 +1,183 @@
+
+Do not use numbers in page headings. Only use numbers less than 10 in section headings.
+No restrictions on using numbers in the body of the text.
+
+### Time and date
+
+Use 24hr time with hours and minutes, but not seconds. Include timezone. For example, 17:00 AEST
+
+More about this in the [Google developer documentation style guide](https://developers.google.com/style/dates-times).
+
+### Spacing after full stops
+
+Refer to [Spacing after full stops in the Markdown syntax guide](markdown-syntax-style.md#spacing-after-full-stops) for more details about this.
+
+### Platform differences
+
+This table summarizes writing style differences across different platforms, distinguishing them from the Buildkite 'Docs'.
+
+|                      | Docs                                                      | Twitter and Blog                                                        | Changelog                                                               |
+|----------------------|-----------------------------------------------------------|-------------------------------------------------------------------------|-------------------------------------------------------------------------|
+| We (as in Buildkite) | No                                                        | Yes                                                                     | Avoid if possible; use 'I' if you need to                               |
+| Exclamation marks    | No                                                        | Yes, although use with restraint, and not more than one in a row        | Yes, although use with restraint, and not more than one in a row        |
+
+### Glossary of notable terms and their spelling
+
+| Word                      | Usage                                                                                            |
+|---------------------------|--------------------------------------------------------------------------------------------------|
+| The Buildkite Agent/agent | When referring to the running process/piece of software as a whole                               |
+| `buildkite-agent`         | When referring to the CLI tool, visually should be presented in a code block                     |
+| Sign up/log in            | The action of signing up (that is, the verb form of these terms)                                                                         |
+| Signup/login              | When referring to a page that enables signing up or to the signup process (that is, the adjective or noun form of these terms)                       |
+| Time out/timeout          | Time out is a verb, timeout is a noun or adjective                                                            |
+| API, SSO, SAML            | Always capitalized                                                                               |
+| GitHub                    | Always capitalized, with an uppercase H in the middle                                            |
```

Comment:
Some general guidance that's missing that you might consider adding is to follow brand capitalization.

Reply:
I might either remove this (because it's covered above in the **Product names (and product features)**), or repurpose this slightly to make it more general about following brand capitalisation, as I suspect you're alluding to.

Reply:
Do the point I mentioned above.

Reply:
Added "premises" to this table and removed GitHub.